src/main/java/umk/mat/pajda/ProjektZespolowy/DTO/ReviewArchiveDTO.java

```diff
@@ -0,0 +1,38 @@
+package umk.mat.pajda.ProjektZespolowy.DTO;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PastOrPresent;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ReviewArchiveDTO{
+
+    @NotNull
+    private Integer id;
+
+    @Size(min=0, max=10)
+    private Integer rating;
+
+    @Size(max=300)
+    private String comment;
```

Comment:
I believe 300 characters might be a too small boundary for a review. Maybe 1000-1500 would be a better choice?

Reply:
No problem, I gave 300, because I've thought that the twitter had 300 characters